app/forms/schools/add_participants/base_wizard.rb

```diff
@@ -473,14 +467,11 @@ def cohort_to_place_participant
         elsif Cohort.within_automatic_assignment_period?
           # true from 1/9 to end of automatic assignment period
           Cohort.current
-        elsif start_term == "summer"
-          Cohort.current
         # we're in the registration window prior to 1/9
-        elsif start_term.in? %w[autumn spring]
-          # we're in the registration window prior to 1/9 and chose autumn or spring the following year
+        elsif induction_start_date.blank?
           Cohort.next
         else
-          # default to now - but should ask the start_term question if not already asked
+          # default to now
           Cohort.current
         end
       end
```

Comment:
I would go for:
```ruby
      def cohort_to_place_participant
        if transfer?
          existing_participant_cohort || existing_participant_profile&.schedule&.cohort
        elsif ect_participant? && induction_start_date.present?
          Cohort.containing_date(induction_start_date).tap do |cohort|
            return Cohort.current if cohort.blank? || cohort.npq_plus_one_or_earlier?
          end
        elsif Cohort.within_automatic_assignment_period?
          Cohort.current
        else
          Cohort.next
        end
      end
```

Reply:
Are there any edge cases where changing the fall-through `else` clause to `Cohort.next` would produce the wrong outcome? I agree it's more succinct this way but I'm nervous that the else clause changes.

Reply:
There is a lot of casuistic about participants and their status at the time of registering them for the first time or transferring them to a new school. 

This ticket is about (at the end of the period of one academic year) sitting participants "potentially compatible" with 2 cohorts (the one about to finish and the new one about to start) into the latest cohort (Cohort.next) so I think the logic I suggest is ok.
But you know, there is always the chance that it is not 100% perfect. 
In any case, if we haven't coped with any specific situation we will know in the testing period or maybe via a support ticket and we'll have to fix it later.

Reply:
Much cleaner as suggested, I've amended with this.